src/org/gk/pro/ProExporterConstants.java

```diff
@@ -0,0 +1,46 @@
+package org.gk.pro;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ProExporterConstants {
+    public static final String plus = "+";
+    public static final String equals = "=";
+    public static final String unknown = "?";
+    public static final String colon = ":";
+    public static final String at = "@";
+    public static final String leftBracket = "[";
+    public static final String rightBracket = "]";
+    public static final String delimiter = "\t";
+    public static final String freeTextDelimiter = "^|^";
+    public static final String ewas = "EWAS";
+    public static final String mod = "=MOD:";
+    public static final String chebi = "=CHEBI:";
+    public static final String uniprot = "=UniProt:";
+    public static final String reactome = "=Reactome:";
+    public static final String stableIdPrefix= "R-HSA-";
+    public static final String insertion = "INSERTION";
+    public static final String deletion = "DELETION";
+    public static final String replaced = "REPLACED";
+    public static final String crosslink = "CROSSLINK";
+
+    public static final String entityType = "Type";
```

Comment:
Since the following constants are not used anywhere else, except for the Columns definition, don't list them here. Just use them directly inside the Columns. You should really make your code simpler. No need for unnecessary lines.

Reply:
Understood! Simplified constants and moved to more appropriate places.